lib/libutils/isoc/include/string.h

```diff
@@ -30,6 +30,7 @@ char *strstr(const char *big, const char *little);
 char *strcpy(char *dest, const char *src);
 char *strncpy(char *dest, const char *src, size_t n);
 char *strrchr(const char *s, int i);
+char* strtok(char *str, const char *delims);
```

Comment:
I'd rather have `strtok_r()` instead, so it can be easily used in thread context.
Style: `char *strtok(...);`.